### DISCUSSION TITLE
Back to School with GitHub Education: Swag Contest

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#65695] [GitHub Education]
![unnamed](https://github.com/community/community/assets/107881423/e1b2d5e2-2316-4c4f-9a25-75eba36ce248)

### Overview
To celebrate Learning Paths on Global Campus, we're running a month-long giveaway contest. All users to complete a Learning Path during the month of September will be entered to win a free piece of GitHub swag. 

### Details & Eligibility
Students who complete any Learning Path (one or both) during the month of September. Students who completed a Learning Path prior to September 1st will be eligible for the Week 1 consideration window. Completing more than one Learning Path does not count as two entries into the contest. Winners selected must redeem their prize by Friday October 13th, 2023. 

### Start here
If you haven’t already, first get verified as a student by filling out the [application here](https://education.github.com/discount_requests/application). Note that it can sometimes take a few days after you’re approved to gain your student benefits. Once you’re all set, log into Global Campus. On your student dashboard, you’ll find one of two Learning Paths to complete. More on Learning Paths can be found [in this Discussion](https://github.com/orgs/community/discussions/64543).

Winners will be chosen randomly and notified via email on the following schedule:

**- Week 1: Eligibility window - On or before September 10th** 
- Selection 9/11 - send email list to BDA
- Notification: starting 9/12

**- Week 2: Eligibility window - September 11 - 17**
- Selection 9/18- send email list to BDA
- Notification starting 9/19

**- Week 3: Eligibility window - September 18 - 24**
- Selection 9/25 - send email list to BDA
- Notification: starting 9/26

**- Week 4: Eligibility window - September 25 - 30**
- Selection 10/2 - send email list to BDA
- Notification: starting 10/3

What might you win? One of these mona figurines!

<img width="1013" alt="Swag Graphic" src="https://github.com/community/community/assets/107881423/f76fcf6d-55b2-4fae-876e-2fd40aa2284b">

**Comment by GabTheGreat25** [comment]
Hope I win! I'm already done with the Learning Path!

**Reply by AnimishY** [reply]
Me Too with one learning path, now on second one! All the best!

**Reply by GabTheGreat25** [reply]
Goodluck with the second one you would absolutely enjoy it!

**Reply by EPro30** [reply]
Where do we find learning paths?

**Reply by Vikash-8090-Yadav** [reply]
On the left side of https://education.github.com/globalcampus/student, you can find the learning path. For a more detailed explanation, follow https://github.com/orgs/community/discussions/64543.

**Reply by jayprakash07** [reply]
Hopefully I win, I've done both learning paths!!!

**Reply by Vikash-8090-Yadav** [reply]
Completed. Learned lots of new things abt Copilot and Cpdespces. I'm excited to see the results and I have gained a lot of new knowledge from this experience.

**Reply by GabTheGreat25** [reply]
Wishing everyone the best of luck! Learning and growth matter most to us.

**Reply by hashamyounis9** [reply]
same here, let's hope for best!
BTW, by completing paths already, have we participated in the contest or we still need to do something to participate?

**Reply by morganersery1** [reply]
@hashamyounis9 There's nothing you need to do to participate other than (at minimum) completing one of the two Learning Paths. As the post says, all who have completed them before this post are entered into the random draw for week 1. No further action is needed on your part and great job completing the paths!

**Reply by UzairHussain193** [reply]
How to check have you completed the learning path or not?

**Reply by Soham-Thodge** [reply]
I just completed the github flow and primer:codespaces path is it good enough for this event?

**Reply by techmannih** [reply]
Here not available for learning yet. can you share a link I will directly on learning path

**Comment by sidowxyz** [comment]
> Hope I win! I'm already done with the Learning Path!

Good luck 🤞

**Reply by 0tieno** [reply]
Hope to be selected. Already done!

**Reply by FARGHALY96** [reply]
How can you verify whether you have finished the learning path or not?
https//:mostasmmer.com

**Comment by AbdulRehmanGHub** [comment]
I opened the [education.github.com/globalcampus/student](url) link but found this message:

Congratulations!
You've completed all available learning paths!
Stay tuned for more learning content from GitHub Global Campus.

What should I do now? As I remember I completed these learning paths 3 to 4 days ago.

**Reply by morganersery1** [reply]
Hello! If you've completed the Learning Paths already, you'll be eligible for entry in week 1. At the end of this week, we'll select at random for all who have completed Learning Paths up to that point. Nothing you need to do. Great job completing the Learning Paths!

**Reply by AbdulRehmanGHub** [reply]
@morganersery1 Thank you for answering my question.

**Comment by azaynul10** [comment]
I have also completed all of the learning paths waiting eagerly for the email. Hope to win it ❤️

**Comment by aman339** [comment]
Hurray.... ❤️ I have completed all available learning paths! 
Thank you for creating these fantastic start-up Learning Paths on Global Campus.
I have learned a lot along with the guide and steps mentioned in the repo. 
Happy Learning to all ❤️

**Comment by Jerome-Jumah** [comment]
Learned a lot from the learning path and loved how code spaces is seamless to use on the browser. Also had fun 😊 deploying the web app to azure which I have never used prior. Can't wait for the next learning path!

**Comment by VedantJadhav258** [comment]
Just got done with the learning path .

**Comment by HVbajoria** [comment]
Loved both the learning paths! ❤️ 
Each and every concept is explained in the most straightforward manner for anyone to understand and complete the steps. Finally, the interactive labs with GitHub Actions make it more engaging and fun.

**Comment by reeveng** [comment]
Back to school with a worse user experience, thanks github! https://github.com/orgs/community/discussions/categories/feed

**Comment by jcgaming-official** [comment]
I just finished my Intro to Github. Goodluck to all🙂 and Thank you Github for amazing learning path and events.

**Reply by reeveng** [reply]
Did the intro include the new feed, because if it did, gotta tell you, better switch to gitlab

**Comment by hritik5102** [comment]
Completed both of the learning paths 🚀

**Reply by techmannih** [reply]
Can you mention those learning path here

**Comment by frederikalzer** [comment]
Education is so important! Thank you Galxe for that.

**Comment by GabTheGreat25** [comment]
Hi! @morganersery1 I am having a problem claiming the Swag Giveaway as my email and personal key isn't working is there any other way to claim it?

**Reply by morganersery1** [reply]
@GabTheGreat25 Congrats on being one of this week's winners. If you're having trouble redeeming, contact our shop support at: [thegithubshop@bdainc.com](mailto:thegithubshop@bdainc.com).

**Reply by AnimishY** [reply]
@GabTheGreat25 Congratulations!

**Reply by Emmarie-Ahtunan** [reply]
Congrats, @GabTheGreat25!

**Reply by aman339** [reply]
Congratulations @GabTheGreat25 ❤️

**Reply by GabTheGreat25** [reply]
Thanks everyone! ❤️

**Reply by GabTheGreat25** [reply]
@morganersery1 how long before the swag prize will be ship? I don't have any ideas because there's no email after I successfully input and submit my details. It's been a week too.

**Reply by antonio-pedro99** [reply]
@GabTheGreat25 I have completed the learning path, how should I claim the swags?

**Comment by pranitrathod** [comment]


**Reply by morganersery1** [reply]
@pranitrathod It's great that you've completing the Learning Paths! All winners are randomly chosen. If you don't end up winning, please know that there are 2 million students who are a part of Global Campus with new students signing up every day. Most students will not win and we only have a limited amount of swag for this contest. But keep an eye out. There will, of course, be more contests in the future!

**Comment by HVbajoria** [comment]
Hello @morganersery1,
Thank you for this amazing learning opportunity! I have received the email forbeing one of the winners but the website isn't working. Even on entering the correct Email it says invalid. 
Could you please help me? 
Thank you

**Reply by morganersery1** [reply]
@HVbajoria Congrats on being one of this week's winners. If you're having trouble redeeming, contact our shop support at: [thegithubshop@bdainc.com](mailto:thegithubshop@bdainc.com).

**Reply by HVbajoria** [reply]
Thank you. It is resolved!

**Comment by LFGaming** [comment]
I got an email today from GitHubEducation@bdainc.com, i have two questions, a, is this the right email? or someone who takes opertunity of this contest. B how am I sure this is real, since ive not signed up to this specific contest, did my Learning paths like a month or two ago, and im extra sceptical because it wants my email or *my personal key* which i find weird.
side note, the fact that this account @morganersery1 has not many public things on her profile makes it even worse for me...

**Reply by morganersery1** [reply]
Hello! I am the Education Partnerships Manager at GitHub. You'll see the Staff tag on my profile. BDA is our shop vendor, so this is the correct email for those notified of winning the contest. All students who completed Learning Paths prior to September 10th were eligible for week 1 of the random draw. You are under no obligation to redeem your prize,  and you do not have to provide a key, just your email in order to redeem.

**Reply by LFGaming** [reply]
Ah thanks, I indeed had not yet seen the staff marker. I'll check it out

**Comment by LucioD3v** [comment]
I hope to win a figure!....

I take advantage of the space to invite those who live in Mexico to sign up for the monthly events we hold:

[GitHub Presente en Español](https://www.meetup.com/es-ES/githublatam/)

**Comment by AnimishY** [comment]
I completed a few more learning paths to participate in this week's giveaway! @morganersery1

**Comment by Sad1616** [comment]
> 
> 
> 
> ![unnamed](https://github.com/community/community/assets/107881423/e1b2d5e2-2316-4c4f-9a25-75eba36ce248)
> 
> ### Overview
> To celebrate Learning Paths on Global Campus, we're running a month-long giveaway contest. All users to complete a Learning Path during the month of September will be entered to win a free piece of GitHub swag. 
> 
> 
> ### Details & Eligibility
> Students who complete any Learning Path (one or both) during the month of September. Students who completed a Learning Path prior to September 1st will be eligible for the Week 1 consideration window. Completing more than one Learning Path does not count as two entries into the contest. Winners selected must redeem their prize by Friday October 13th, 2023. 
> 
> ### Start here
> If you haven’t already, first get verified as a student by filling out the [application here](https://education.github.com/discount_requests/application). Note that it can sometimes take a few days after you’re approved to gain your student benefits. Once you’re all set, log into Global Campus. On your student dashboard, you’ll find one of two Learning Paths to complete. More on Learning Paths can be found [in this Discussion](https://github.com/orgs/community/discussions/64543).
> 
> Winners will be chosen randomly and notified via email on the following schedule:
> 
> **- Week 1: Eligibility window - On or before September 10th** 
> - Selection 9/11 - send email list to BDA
> - Notification: starting 9/12
> 
> **- Week 2: Eligibility window - September 11 - 17**
> - Selection 9/18- send email list to BDA
> - Notification starting 9/19
> 
> **- Week 3: Eligibility window - September 18 - 24**
> - Selection 9/25 - send email list to BDA
> - Notification: starting 9/26
> 
> **- Week 4: Eligibility window - September 25 - 30**
> - Selection 10/2 - send email list to BDA
> - Notification: starting 10/3
> 
> What might you win? One of these mona figurines!
> 
> <img width="1013" alt="Swag Graphic" src="https://github.com/community/community/assets/107881423/f76fcf6d-55b2-4fae-876e-2fd40aa2284b">
> 
> 
> 
> ### A suggestion...(optional)
> 
> _No response_

**Reply by theinfinox** [reply]
Great <3

**Comment by LucioD3v** [comment]
I'm already done with the Learning Path!


![image](https://github.com/community/community/assets/6353852/d6235fed-2a37-4645-993f-362fb172c41b)

[https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-LucioMSP](https://github.com/Exp-Intro-to-GitHub-Flow-Cohort-1/series-intro-to-github-flow-LucioMSP)

**Comment by hxlog** [comment]
I am also just finishing Learning Path, good luck to all 🎉, cheers!  🥳

**Comment by vivek9758** [comment]
Completed all learning paths 🎉 
So ? What next process win the swag ?
@morganersery1

**Reply by AnimishY** [reply]
I don't think there is any other step in this.

**Comment by TauqeerAhmad5201** [comment]
This is definitely an amazing hands-on tutorial for beginners. It is a good way of using workflows to educate as well. It improves accuracy because we have already created our path using workflow. [👍](https://emojipedia.org/thumbs-up)

**Comment by yeskaydee** [comment]
@morganersery1 
--> Selection 9/18- send email list to BDA <--

what does this BDA means??

**Reply by AnimishY** [reply]
BDA is their shop vendor. They share the list of winners with BDA

**Reply by yeskaydee** [reply]
@AnimishY  oh so they share the selected list to their vendor...got it thanks man

**Reply by yeskaydee** [reply]
@AnimishY btw how can we see if we have completed the challenge and were considered for the pool of random selection?

**Comment by jcgaming-official** [comment]
I didn't win lol. Just give me swag

**Comment by devil-1964** [comment]
🤞

**Reply by devil-1964** [reply]
![image](https://github.com/community/community/assets/98086933/43cf357f-f165-45c9-9b75-a469925e8ce7)

**Comment by Emmarie-Ahtunan** [comment]
My fingers are crossed to be chosen this week or next. 
I'd love to win a Monacat figurine! 
:octocat: ❤️ :octocat:

**Comment by aman339** [comment]
Yes absolutely, I'm also very excited to be selected in the current or upcoming week. 🚀🥳
I would love to win some very cute Monacat figurines too.  :octocat:  ❤️
Happy Learning to all ❤️

**Comment by yeskaydee** [comment]
wow never knew codespaces is so good

**Comment by SykChw** [comment]
Just finished Learning Path, good luck to you all, cheers! 🎉

**Comment by yeskaydee** [comment]
@morganersery1 did't recieve any email 🥲

**Reply by morganersery1** [reply]
Hi @yeskaydee  All winners are randomly chosen. If you did not receive an email, it means that you were not one of the randomly chosen winners. Please know that there are 2 million students who are a part of Global Campus with new students signing up every day and only 10 winners per week throughout September. Most students will not win and we only have a limited amount of swag for this contest. But there will be more contests in the future and more opportunities to win.

**Reply by selcukemiravci** [reply]
Definitely, thanks a bunch for putting these learning paths for beginners. Even though they are very short, I believe they will be very beneficial to some of the students especially who are just getting familiar with coding or Github in general.

I've also shared a short feedback on two of these workflows as a student leader and a student who has done developer advocacy in the past at a top blockchain firm! If you are interested, you can read more in [here](https://github.com/orgs/community/discussions/64543#discussioncomment-7323396) and I hope it helps to make future workflows more and more engaging! Definitely looking forward to future ones! 🚀

**Comment by LFGaming** [comment]
Hi, I got a email saying:
`Here is your tracking for the GitHub Back to School Contest ...`
but there is no tracking link in there, how does that work?

**Reply by SerenaMullen** [reply]
@LFGaming I will look into you not receiving your tracking and get back you once I have an answer for you

**Comment by LFGaming** [comment]
I've received mine!!
It is an amazing little statue, thank you so much.

FYI, the whiskers are not that strong so look out for that ;)

**Comment by GCC1111** [comment]
Safari reset

**Comment by miniMaddy** [comment]
I received the figurine today. It's simple amazing. 
Thank GitHub!!

**Reply by RedStar071** [reply]
How long did you have to wait to receive the package containing the figurine?
